Why: Install lrexlib: it is a private dependency of wslua, but must be linked explicitly into the static wasm build.

diff --git a/epan/wslua/lrexlib/CMakeLists.txt b/epan/wslua/lrexlib/CMakeLists.txt
index 5f9beb08d9becc918c95a14568a23379b7ade574..0888ac5e1483bb880538745821b77b4ddd3b0d5d 100644
--- a/epan/wslua/lrexlib/CMakeLists.txt
+++ b/epan/wslua/lrexlib/CMakeLists.txt
@@ -29,3 +29,17 @@ add_compile_definitions(
 	VERSION=\"2.9.2\"
 	PCRE2_CODE_UNIT_WIDTH=8
 )
+
+set_target_properties(lrexlib PROPERTIES
+	PREFIX "lib"
+	LINK_FLAGS "${WS_LINK_FLAGS}"
+	FOLDER "DLLs"
+	INSTALL_RPATH "${LIBRARY_INSTALL_RPATH}"
+)
+
+install(TARGETS lrexlib
+	EXPORT WiresharkTargets
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
\ No newline at end of file
